nextcord/role.py

```diff
@@ -261,6 +263,8 @@ def _update(self, data: RolePayload) -> None:
         except KeyError:
             self.tags = None
 
+        self._flags = data.get("flags", 0)
```

Comment:
```suggestion
        self._flags: int = data.get("flags", 0)
```
like every other attribute here